pkgs/tools/misc/fontforge/default.nix

```diff
@@ -30,6 +30,12 @@ stdenv.mkDerivation rec {
       url = "https://salsa.debian.org/fonts-team/fontforge/raw/76bffe6ccf8ab20a0c81476a80a87ad245e2fd1c/debian/patches/0001-add-extra-cmake-install-rules.patch";
       sha256 = "u3D9od2xLECNEHhZ+8dkuv9818tPkdP6y/Tvd9CADJg=";
     })
+    # Fix segmentation fault with some fonts.
+    # This is merged and should be present in the next release.
+    (fetchpatch {
```

Comment:
:+1: for the patch.

Reply:
If this is causing the mass rebuild than it would be good we can do a PR that targets staging with it and wait until it is in master.

Reply:
This patch has been upstreamed. We can drop this commit from the PR.